### PR TITLE
ENH: make a latex builder base class

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks for putting this in @CJ-Wright! I think that there is also the publists and gradesbuilders that also should be based on the new latexbuilder

File: regolith/tools.py [hunk 1]
```diff
@@ -225,10 +226,20 @@ def awards_grants_honors(p):
         }
         aghs.append(d)
     for x in p.get('service', []) + p.get('honors', []):
-        d = {'description': latex_safe(x['name']),
-             'year': x['year'],
-             '_key': date_to_float(x['year'], x.get('month', 0)),
-             }
+        print(x)
```
Comment (scopatz) [inline regolith/tools.py]: probably don't need this print.

File: regolith/tools.py [hunk 2]
```diff
@@ -216,6 +216,7 @@ def awards_grants_honors(p):
         The person entry
     """
     aghs = []
+    print(p['name'])
```
Comment (scopatz) [inline regolith/tools.py]: probably don't need this sprint

**Comment by CJ-Wright** [comment]
Should we have a folder for all the builders?

**Comment by CJ-Wright** [comment]
@scopatz ready for another round.

**Comment by scopatz** [comment]
Eh I am a flat-is-better person in general, but if you think it is time to have a builders subfolder, that is fine too

**Comment by scopatz** [comment]
This looks good to go to me barring the point you made

**Comment by CJ-Wright** [comment]
I'll open an issue for that (no need to block this)

**Pull request merged**